File: Ejercicios Propuestos 3/Ejercicio 1/Ejercicio 1.jl
suma = 0

print("Ingrese la cantidad de elementos que tendrá el vector: ")
cantidad = parse(Int, readline())

vectorNumeros = Vector{Int}(undef, cantidad)

println("Ingrese los números enteros uno por uno:")

for i in 0:(cantidad - 1)
    print("Valor para la posición ", i, ": ")
    
    # Guardamos en i + 1 (para usar 1, 2, 3...)
    vectorNumeros[i + 1] = parse(Int, readline())
    
    global suma = suma + vectorNumeros[i + 1]
end

println("La suma total de los elementos del vector es: ", suma)

File: Ejercicios Propuestos 3/Ejercicio 1/Ejercicio1.java
import java.util.Scanner;

public class Ejercicio1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // Definir cantidad, i, suma Como Entero
        int cantidad, i, suma;
        suma = 0;

        System.out.println("Ingrese la cantidad de elementos que tendrá el vector:");
        cantidad = sc.nextInt();

        // Dimension vectorNumeros[cantidad]
        // Creamos el arreglo (vector) en Java con el tamaño ingresado
        int[] vectorNumeros = new int[cantidad];

        // Lectura de elementos y cálculo de la suma
        System.out.println("Ingrese los números enteros uno por uno:");

        // Para i <- 0 Hasta cantidad - 1 Con Paso 1 Hacer
        for (i = 0; i <= cantidad - 1; i++) {
            System.out.println("Valor para la posición " + i + ":");
            vectorNumeros[i] = sc.nextInt();

            suma = suma + vectorNumeros[i];
        }

        // Resultado final
        System.out.println("La suma total de los elementos del vector es: " + suma);

        sc.close();
    }
}